src/components/listing/FileMetadata.vue

```diff
@@ -104,13 +107,23 @@ export default class FileMetadata extends Vue {
         }
         return
       }
+      // TODO: figure out a cleaner way?
```

Comment:
can be ternary `this.titleEditable = (mutation.payload === ProcessStatus.Executing) ?  false : true`

Reply:
good catch